README.md

```diff
@@ -7,3 +7,140 @@ Requirements
 ------------
 
 - [Django](https://docs.djangoproject.com) 2.2.x
+- [Django AppSettings](https://pypi.org/project/django-app-settings/)
+- [lxml](https://pypi.org/project/lxml/)
+- [xmlsec](https://pypi.org/project/xmlsec/)
+- [Apache Ignite Python client](https://pypi.org/project/pyignite/) (optional)
+
+Specific Proxy Service
+----------------------
+
+### Usage
+
+To use eIDAS Proxy Service, adjust Django settings:
+
+* Set up  Django template engine and static file.
+* Add `'eidas_node.proxy_service.apps.ProxyServiceConfig'` to `INSTALLED_APPS`.
+* Set `ROOT_URLCONF` to `'eidas_node.proxy_service.urls'` or include it in your URL configuration.
+* Provide mandatory configuration options.
+
+```py
+"""
+Development settings for eidas_node Django app.
+
+DO NOT USE IN PRODUCTION!
+
+See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+"""
+
+# Security
+SECRET_KEY = 'secret
+DEBUG = True
+ALLOWED_HOSTS = ['*']
+
+# Application definition
+INSTALLED_APPS = [
+    'django.contrib.staticfiles',
+    'eidas_node.proxy_service.apps.ProxyServiceConfig',
+]
+MIDDLEWARE = [
+    'django.middleware.common.CommonMiddleware',
+]
+ROOT_URLCONF = 'eidas_node.proxy_service.urls'
+
+STATIC_ROOT = '/var/www/eidas-node-proxy-service/static'
+STATIC_URL = '/static/'
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.messages.context_processors.messages',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.template.context_processors.i18n',
+            ]
+        }
+    }
+]
+
+# eIDAS Proxy Service
+PROXY_SERVICE_REQUEST_TOKEN = ...
+PROXY_SERVICE_RESPONSE_TOKEN = ...
+PROXY_SERVICE_LIGHT_STORAGE = ...
+PROXY_SERVICE_IDENTITY_PROVIDER = ....
+PROXY_SERVICE_EIDAS_NODE = ...
+```
+
+### Settings
+
+#### `PROXY_SERVICE_REQUEST_TOKEN`
+
+Settings of **a light token corresponding to an incoming light request**. A dictionary with following items:
+
+* `HASH_ALGORITHM` (optional, default `'sha256'`): A hash algorithm used for token digest.
+* `SECRET` (required): A token secret shared with eIDAS node.
+* `ISSUER` (required): An issuer of the light token.
+* `LIFETIME` (optional, default `10`): A lifetime of the light token in minutes until is its considered expired. Set to
```

Comment:
We tend to put one sentence per line.

Reply:
Without line wrapping at 120 characters?

Reply:
Yes.

Reply:
I've reformatted the file.